contracts/interfaces/GsnTypes.sol

```diff
@@ -10,6 +10,8 @@ interface GsnTypes {
         uint256 baseRelayFee;
         address relayWorker;
         address paymaster;
+        bytes32 paymasterData;
```

Comment:
bytes

bytes32 may be not enough for some use-cases

Reply:
if you want more signed data that bytes32, then put the rest into approvalData, and just put a hash here. 
No need to make the basic structure overly complex.

Reply:
No need to make client logic overly complex. 
It makes total sense to have bytes and it has zero cost, code or otherwise.

Reply:
fixed